lib/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -110,25 +110,6 @@ namespace {
         CommitState() = delete;
     };
 
-    //! Helper utility function to get number of draw items required for given representation
-    size_t _GetNumDrawItemsForDesc(const HdMeshReprDesc& reprDesc)
-    {
-        // By default, each repr desc item maps to 1 draw item
-        size_t numDrawItems = 1;
-
-        // Different representations may require different number of draw items
-        // See HdSt for an example.
-        switch (reprDesc.geomStyle) {
-        case HdMeshGeomStyleInvalid:
-            numDrawItems = 0;
-            break;
-        default:
-            break;
-        }
-
-        return numDrawItems;
```

Comment:
I guess for now it is always 0 or 1, we can always bring it back if we need it.

Reply:
Yes. Having only one for each repr desc is easier to manage, until we find it necessary to break this assumption.